Documentation/ceph-object.md

```diff
@@ -285,15 +285,15 @@ metadata:
 spec:
   ports:
   - name: rgw
```

Comment:
Agreed that nodeport is a bad example, we actually have three examples in the [manifests folder](https://github.com/rook/rook/blob/master/deploy/examples) for ingress, loadbalancer, and nodeport. There are a few other aspects to this section that need to be updated as well instead of just the port, such as not mentioning NodePort in the first paragraph. How update updating this section to match the [loadbalancer example](https://github.com/rook/rook/blob/master/deploy/examples/dashboard-loadbalancer.yaml)?